src/sysfiles.c

```diff
@@ -597,12 +597,8 @@ void syWinPut (
     if ( ! SyWindow || 4 <= fid )
         return;
 
-    /* get the file descriptor                                             */
-    if ( fid == 0 || fid == 2 )  fd = syBuf[fid].echo;
-    else                         fd = syBuf[fid].fp;
-
```

Comment:
Is there a particular benefit in renaming `writeandcheck` to `echoandcheck`, and changing its `fd` parameter to `fid`, as opposed to just replacing the above few lines by `fd = syBuf[fid].echo`? That does the same, doesn't it?

Reply:
It's not that useful here, but it will be useful for my later patch, as it means that places where we have raw file handles are much smaller (and all those places need to be changed when we add gzip support).